api/src/test/java/io/cloudevents/types/TimeTest.java

```diff
@@ -46,18 +46,18 @@ void testParseDateOffset() {
 
     @Test
     void testSerializeDateOffset() {
-        assertThat(Time.writeTime(ZonedDateTime.of(
+        assertThat(Time.writeTime(OffsetDateTime.of(
             LocalDateTime.of(2020, 8, 3, 18, 10, 0, 0),
-            ZoneId.of("Europe/Rome")
-        ))).isEqualTo("2020-08-03T18:10:00+02:00");
+            ZoneOffset.ofHours(2)
+        ))).isEqualTo("2020-08-03T18:10+02:00");
     }
 
     public static Stream<Arguments> parseDateArguments() {
         return Stream.of(
-            Arguments.of("1985-04-12T23:20:50.52Z"),
-            Arguments.of("1990-12-31T23:59:00Z"),
-            Arguments.of("1990-12-31T15:59:00-08:00"),
-            Arguments.of("1937-01-01T12:00:27.87+00:20")
+            Arguments.of("1985-04-12T23:20:50.520Z"),
+            Arguments.of("1990-12-31T23:59Z"),
```

Comment:
@slinkydeveloper is this a valid RFC3339 date if there are no seconds?

Reply:
I was looking at the abnf, it seems like i made a mistake here

```
   partial-time    = time-hour ":" time-minute ":" time-second
                     [time-secfrac]
   full-date       = date-fullyear "-" date-month "-" date-mday
   full-time       = partial-time time-offset
```

partial-time has time-second not optional